mariner-installer/build/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+echo "BRANCH: $BRANCH"
+cd $WORKSPACE/mariner-installer
+
+. /etc/os-release
```

Comment:
do we do this anywhere else? (not sure why are we sourcing here).

Jenkins should slap a bunch of environment variables for you, so not sure I understand why the need to source this. Are the env vars supplied not sufficient?